.eslintrc.js

```diff
@@ -8,5 +8,18 @@ module.exports = {
 
   rules: {
     'space-before-function-paren': 0
-  }
+  },
+
+  globals: {
```

Comment:
Is it really necessary to add Jest globals to eslintrc globally? It also allows to use 'em in real code

Reply:
ok, I have removed this code from project